backends/advanced/src/advanced_omi_backend/plugins/homeassistant/plugin.py

```
@@ -596,3 +599,94 @@ async def _parse_command_hybrid(self, command: str) -> Optional['ParsedCommand']
+            if str(test_result).strip() != "2":
+                return {
+                    "success": False,
+                    "message": f"Unexpected template result: {test_result}",
+                    "status": "error"
+                }
+
+            # Try to fetch entities count for additional info
+            try:
+                entities = await mcp_client.get_all_entities()
+                entity_count = len(entities)
+            except Exception:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)